di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/config/OrchestratorConfig.java

```diff
@@ -11,26 +11,31 @@ public class OrchestratorConfig {
             getConfigValue("ORCHESTRATOR_CLIENT_ID", "orchestrator");
     public static final String ORCHESTRATOR_REDIRECT_URL =
             getConfigValue("ORCHESTRATOR_REDIRECT_URL", "http://localhost:4500/callback");
-    public static final String ORCHESTRATOR_CLIENT_SIGNING_KEY =
+    public static final String ORCHESTRATOR_SIGNING_JWK =
             getConfigValue(
-                    "ORCHESTRATOR_CLIENT_SIGNING_KEY",
+                    "ORCHESTRATOR_SIGNING_JWK",
                     // This is a test key used for local development
-                    "MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgOXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthWhRANCAAQT1nO46ipxVTilUH2umZPN7OPI49GU6Y8YkcqLxFKUgypUzGbYR2VJGM+QJXk0PI339EyYkt6tjgfS+RcOMQNO"); // pragma: allowlist secret
-    public static final String ORCHESTRATOR_DEFAULT_JAR_ENCRYPTION_PUBLIC_KEY =
+                    "{\"kty\":\"EC\",\"d\":\"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU\",\"crv\":\"P-256\",\"x\":\"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM\",\"y\":\"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04\"}"); // pragma: allowlist secret
```

Comment:
Do these default keys need kid/use as well?

Reply:
I don't think these are actually ever used. I'll add them in though just in case

Reply:
I was going to say local running, but looks like we set those explicitly as well :shrug: